fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CreateTableAnalyzer.java

```diff
@@ -93,19 +94,26 @@ private static String analyzeEngineName(String engineName, String catalogName) {
                     throw new SemanticException("Unknown engine name: %s", engineName);
                 }
             }
-        } else {
-            String catalogType = GlobalStateMgr.getCurrentState().getCatalogMgr().getCatalogType(catalogName);
+        }
+
+        String catalogType = GlobalStateMgr.getCurrentState().getCatalogMgr().getCatalogType(catalogName);
+        if (catalogType.equalsIgnoreCase(UNIFIED)) {
             if (Strings.isNullOrEmpty(engineName)) {
-                if (!catalogType.equalsIgnoreCase(ICEBERG) && !catalogType.equalsIgnoreCase(HIVE)) {
-                    throw new SemanticException("Currently doesn't support creating tables of type " + catalogType);
-                }
-                return catalogType;
-            } else if (!engineName.equalsIgnoreCase(catalogType)) {
-                throw new SemanticException("Can't create %s table in the %s catalog", engineName, catalogType);
-            } else {
-                return engineName;
+                throw new SemanticException("Create table in unified catalog requires engine clause " +
```

Comment:
plz give a example how to create a table in unified catalog

Reply:
add a sql in description